src/lib/commands/version.command.ts

```diff
@@ -12,5 +13,6 @@ export const version: Command<any, void> = () => {
   console.log(`ng-packagr:            ` + require('../../package.json').version);
   console.log(`@angular/compiler:     ` + COMPILER_VERSION.full);
   console.log(`rollup:                ` + ROLLUP_VERSION);
+  console.log(`ts-node:               ` + TS_NODE_VERSION);
```

Comment:
I created this PR to display the `ts-node` version for debugging of issues like #1867, then I realized that ts-node is imported only by the integration tests.

Closing.